index.html

```diff
@@ -405,17 +409,16 @@ <h6>Skills</h6>
                 placeholder="write your message here"
               ></textarea>
             </li>
-
-            <a class="button form-button" href="#" type="submit"
-              >Get in touch</a
-            >
+            <button class="button form-button" onclick="test()" type="submit">
+              Get in Touch
+            </button>
+            <span class="alert"></span>
```

Comment:
[OPTIONAL] Your error message displays close to the submit button. But I think it should be closer, probably on top of the button